### PR TITLE
Catch KeyError('MODULE_YAML_PATH') when initializing the app

### PR DESCRIPTION
**Pull request by gregsako**
Catch KeyError('MODULE_YAML_PATH') when initializing the app, and throw a more helpful error.

### CODE REVIEW

**Comment by jonparrott** [comment]
@gregsako when can this occur? The runtime sort of depends on the presence of app.yaml, yeah?

File: appengine-vmruntime/vmruntime/wsgi.py [hunk 1]
```diff
@@ -43,8 +43,19 @@
...
+    appinfo = wsgi_config.get_module_config(
+        wsgi_config.get_module_config_filename())
+except KeyError as e:
+  if e.args[0] == 'MODULE_YAML_PATH':
+    raise RuntimeError('An error occured when initializing the runtime. If you '
+        'are using \'env: flex\', this may be because you forgot to set '
+        '\'enable_app_engine_apis: true\' under beta_settings in app.yaml.')
+  else:
+    raise
+except:
```
Comment (jonparrott) [inline appengine-vmruntime/vmruntime/wsgi.py]: This block isn't needed.

**Comment by jonparrott** [comment]
Oh nevermind, I see.

**Comment by jonparrott** [comment]
@gregsako the runtime image will need to be rebuilt. If you need help with that let me know.

**Comment by gregsako** [comment]
Thanks!

**Pull request merged**